ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -58,6 +62,19 @@ objects:
     - name: cluster-profile
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
+    - name: license-pull-secret-aws
```

Comment:
this is for devs to be able to run locally.. we can put this in an if_local_dev block or something.. or not.. if we don't want devs running this (separate issue than this PR's original `run smoke tests in aws-e2e job`  )

Reply:
Yes please let's break out the local dev work

Reply:
yup, will do